src/martian/testing.py

```diff
@@ -62,5 +62,15 @@ def __init__(cls, classname, bases, dict_):
         fake_import(cls)
         return type.__init__(cls, classname, bases, dict_)
 
-class FakeModule(object):
-    __metaclass__ = FakeModuleMetaclass
+
+if sys.version_info[0] < 3:
+    class FakeModuleObject(object):
+        pass
+    
+    class FakeModule(object):
+        __metaclass__ = FakeModuleMetaclass
+else:
+    from martian.testing_compat3 import FakeModule
+    from martian.testing_compat3 import FakeModuleObject
+
+
```

Comment:
[six.with_metaclass](https://pythonhosted.org/six/#six.with_metaclass) would come in handy here.